sunbird_service_api_test/src/test/java/org/sunbird/integration/test/org/CreateOrganisationTest.java

```diff
@@ -11,54 +11,68 @@
 
 public class CreateOrganisationTest extends BaseCitrusTest {
 
-  public static final String TEST_NAME_CREATE_SUB_ORG_FAILURE_WITHOUT_NAME =
-      "testCreateSubOrgFailureWithoutName";
-  public static final String TEST_NAME_CREATE_SUB_ORG_FAILURE_WITH_PROVIDER_WITHOUT_EXTERNAL_ID =
-      "testCreateSubOrgFailureWithProviderWithoutExternalId";
-  public static final String TEST_NAME_CREATE_SUB_ORG_FAILURE_WITH_INVALID_LOCATION_CODE =
-      "testCreateSubOrgFailureWithInvalidLocationCode";
-  public static final String TEST_NAME_CREATE_SUB_ORG_FAILURE_WITH_INVALID_CHANNEL =
-      "testCreateSubOrgFailureWithInvalidChannel";
+	public static final String TEST_NAME_CREATE_SUB_ORG_FAILURE_WITHOUT_NAME =
+			"testCreateSubOrgFailureWithoutName";
+	public static final String TEST_NAME_CREATE_SUB_ORG_FAILURE_WITH_PROVIDER_WITHOUT_EXTERNAL_ID =
+			"testCreateSubOrgFailureWithProviderWithoutExternalId";
+	public static final String TEST_NAME_CREATE_SUB_ORG_FAILURE_WITH_INVALID_LOCATION_CODE =
+			"testCreateSubOrgFailureWithInvalidLocationCode";
+	public static final String TEST_NAME_CREATE_SUB_ORG_FAILURE_WITH_INVALID_CHANNEL =
+			"testCreateSubOrgFailureWithInvalidChannel";
+	public static final String TEST_NAME_CREATE_ROOT_ORG_FAILURE_WITH_EXISTING_CHANNEL="testCreateRootOrgFailureWithExistingChannel";

```

Comment:
Remove tests such as this which have dependency on another entity to be created (e.g. org).